src/modules/75-cf/RouteDestinations.tsx

```diff
@@ -249,9 +250,19 @@ const CFRoutes: FC = () => {
           ...targetPathProps
         })}
         exact
-        pageName={FFM_1827 ? PAGE_NAME.TargetDetailPage : PAGE_NAME.LegacyTargetDetailPage}
+        pageName={
+          FFM_1827 || FFM_6665_FF_MFE_Target_Detail ? PAGE_NAME.TargetDetailPage : PAGE_NAME.LegacyTargetDetailPage
+        }
       >
-        <FFGitSyncProvider> {FFM_1827 ? <TargetDetailPage /> : <LegacyTargetDetailPage />} </FFGitSyncProvider>
+        <FFGitSyncProvider>
+          {FFM_1827 && !FFM_6665_FF_MFE_Target_Detail ? (
+            <TargetDetailPage />
+          ) : !FFM_1827 && FFM_6665_FF_MFE_Target_Detail ? (
+            <FFUIApp />
+          ) : (
+            <LegacyTargetDetailPage />
+          )}
```

Comment:
We can assume that the MFE will take precedence here, regardless of other flags. As-is, if we turn on the MFE for an env where the newer page is enabled (which is everywhere, for all customers, except with us in UAT), they'll actually get the legacy version. Maybe try something like this...
```suggestion
          {FFM_6665_FF_MFE_Target_Detail
            ? <FFUIApp />
            : FFM_1827 ? <TargetDetailPage /> : <LegacyTargetDetailPage />
          }
```

Reply:
fair enough, updated now 👍